src/lib/company-emissions/companyEmissionsCalculator.ts

```diff
@@ -41,25 +53,41 @@ function calculateEmissionTotals(currentPeriod: any, previousPeriod: any) {
     scope1: currentScope1,
     scope2: currentScope2,
     scope3: currentScope3,
+    scope1And2: currentScope1And2,
   } = currentPeriod.emissions || {}
   const {
     scope1: previousScope1,
     scope2: previousScope2,
     scope3: previousScope3,
+    scope1And2: previousScope1And2,
   } = previousPeriod.emissions || {}
 
-  // Compare Scope 1 emissions
-  if (currentScope1 && previousScope1) {
-    adjustedCurrentTotal += currentScope1?.total ?? 0
-    adjustedPreviousTotal += previousScope1?.total ?? 0
-  }
+  // Calculate scope 1+2 totals for each period independently
```

Comment:
I would love it if we can make a small internal function `calculatePeriodTotal(scope1, scope2)` that we can call for both the current and prevoius periods below to get the current/previous Scope1And2Total, to avoid repeating the code for both periods here and make it more readable  :)